postgres/README.md

```diff
@@ -189,9 +189,13 @@ primary key columns to be `NOT NULL`. However, in PostgreSQL, a primary key is a
 combination of `NOT NULL` and `UNIQUE`, and so primary key columns from
 PostgreSQL will be mapped to Spanner columns that are both primary keys and `NOT NULL`.
 
-### Foreign Keys and Default Values
+### Foreign Keys
 
-Spanner does not currently support foreign keys or default values. We drop these
+Spanner requires foreign key constraint name to be unique accross database while in postgres it has to be unique for a table only, so we map the foreign key constraint name to a unique constraint name for spanner. spanner doesn't support `ON DELETE` and `ON UPDATE` rules in foreign keys, so we skip those rules. We handle foreign keys using ALTER TABLE statement after all the tables are created, this helps us avoid cyclic dependencies in foreign keys.
```

Comment:
I'd drop the comment about using ALTER TABLE: I don't think it is needed (it's kind of an implementation detail), and in any case Postgre and MySQL dump formats do essentially the same thing so it's not a surprise.

Suggest rephrasing as:

The tool maps PostgreSQL foreign key constraints into Spanner foreign key constraints, and
preserves constraints names where possible. Note that Spanner requires foreign key
constraint names to be globally unique (within a database), but in postgres they only
have to be unique for a table, so we add a uniqueness suffix to a name if needed.
Spanner doesn't support ON DELETE and ON UPDATE actions, so we drop these.